app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
```

Comment:
Leaving in ```console``` will cause a syntax error on certain older browsers (such as IE) and should be removed. As a workaround, I suggest defining your own debug function that checks for the existence of ```console```